src/client/pages/OfferNew/Introduction/HeroOfferDetails.tsx

```diff
@@ -47,8 +47,10 @@ const Address = styled.div`
 
 export const HeroOfferDetails: React.FC<Props> = ({ offerData }) => {
   const { person, quotes } = offerData
-  const numberCoInsured = getHouseholdSize(quotes[0].quoteDetails) - 1
+  const numberCoInsured = person.householdSize - 1
```

Comment:
Nice! 🌟 I think we do this unnecessary thing in more places actually, so that's something that can be changed when one runs into that